picard/ui/options/interface_theme.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+# Copyright (C) 2019 Laurent Monin
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+from functools import partial
+
+from PyQt5 import (
+    QtCore,
+    QtGui,
+    QtWidgets,
+)
+
+from picard import config
+
+from picard.ui.options import (
+    OptionsPage,
+    register_options_page,
+)
+from picard.ui.ui_options_interface_theme import Ui_InterfaceThemeOptionsPage
+from picard.ui.util import picard_theme
+
+
+class ColorButton(QtWidgets.QPushButton):
+
+    color_changed = QtCore.pyqtSignal(str)
+
+    def __init__(self, initial_color=None, parent=None):
+        super().__init__('    ', parent=parent)
+
+        color = QtGui.QColor(initial_color)
+        if not color.isValid():
+            color = QtGui.QColor("black")
+        self.color = color
+        self.clicked.connect(self.open_color_dialog)
+        self.update_color()
+
+    def update_color(self):
+        self.setStyleSheet("QPushButton { background-color: %s; }" % self.color.name())
+
+    def open_color_dialog(self):
+        dlg = QtWidgets.QColorDialog()
+        new_color = dlg.getColor(self.color, title=_("Choose a color"), parent=self)
+
+        if new_color.isValid():
+            self.color = new_color
+            self.update_color()
+            self.color_changed.emit(self.color.name())
+
+
+def deleteItemsOfLayout(layout):
+    if layout is not None:
+        while layout.count():
+            item = layout.takeAt(0)
+            widget = item.widget()
+            if widget is not None:
+                widget.setParent(None)
+            else:
+                deleteItemsOfLayout(item.layout())
+
+
+class InterfaceThemeOptionsPage(OptionsPage):
+
+    NAME = "interface_theme"
+    TITLE = N_("Theme")
```

Comment:
The thing is I'm thinking about adding more than just colors configuration here (like hardcoded fonts).

Reply:
I saw you renamed it to "Colors", and I really like this better than "Themes" to not cause confusion. But I see your point. Maybe we could come up with a different name in case other options get integrated, something like "Appearance" maybe?